core/src/main/java/com/graphhopper/reader/dem/ElevationInterpolation.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for
+ *  additional information regarding copyright ownership.
+ *
+ *  GraphHopper GmbH licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
+ *  compliance with the License. You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.graphhopper.reader.dem;
+
+/**
+ * This enum defines the interpolation method to use.
+ */
+public enum ElevationInterpolation {
+    NONE("none"), BILINEAR("bilinear");
+
+    private final String name;
+
+    ElevationInterpolation(String name) {
+        this.name = name;
+    }
+
+    @Override
+    public String toString() {
+        return name;
+    }
+
+    public static ElevationInterpolation find(String name) {
+        if (name == null || name.isEmpty())
+            return NONE;
+
+        for (ElevationInterpolation interpolation : values()) {
+            if (interpolation.name().equalsIgnoreCase(name)) {
+                return interpolation;
+            }
+        }
+
+        return NONE;
+    }
```

Comment:
I would keep it simple for now and only use the two strings :) ? Or why did you choose to introduce this enum?

Reply:
Good call, that's over-engineering for now. Since we only have one interpolation method so far, just a boolean is good.  I think we should keep the config as `graph.elevation.interpolation: bilinear` so it's clear what it's doing and so configs don't have to change if we add other interpolation methods later.